### PR TITLE
Legal Updates

### PR DESCRIPTION
**Pull request by MatthewKhouzam**
Add:
* License
* Contriubting
* Notice

### CODE REVIEW

**Comment by MatthewKhouzam** [comment]
Need to fix contributing

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) [year] [fullname]
```
Comment (marcdumais-work) [inline LICENSE.md:3]: The license is included in the initial contribution PR, with year and copyright holder specified. So I think you could remove this file from the PR, or fill-on year/fullname and remove it from the other PR.

Comment (MatthewKhouzam) [reply LICENSE.md:3]: Whoopsiedoodles! I'll fix after #3 is merged, thanks for the catch!

**Comment by MatthewKhouzam** [comment]
I think we need to fix contributing and notices...

**Pull request closed without merging**